src/test/java/seedu/address/testutil/PersonUtil.java

```diff
@@ -20,7 +21,7 @@
     /**
      * Returns an add command string for adding the {@code person}.
      */
-    public static String getAddCommand(Person person) {
+    public static String getAddCommand(Person person, PersonType personType) {
```

Comment:
May not need the second `personType` argument since it's not used in this method. The differentiation between a `Student` and `Tutor` can be done in `getAddCommand()` in `StudentUtil` and `TutorUtil` respectively.